ElementX/Sources/Screens/Timeline/View/TimelineItemViews/AudioRoomTimelineView.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 import SwiftUI
 
 struct AudioRoomTimelineView: View {
+    @EnvironmentObject private var context: TimelineViewModel.Context
```

Comment:
Should probably use the non-observed one for all of these additions, seeing as we're only using it to send actions and don't need to react to it.
```suggestion
    @Environment(\.timelineContext) private var context
```

Reply:
Very good point! 👍

Reply:
One day we'll switch to Observable and everything will be beautiful 😄